pandas/io/excel/_base.py

```diff
@@ -781,6 +778,19 @@ def close(self):
         return self.save()
 
 
+def _is_ods_stream(stream):
```

Comment:
can you type this function inputs & outputs & add a doc-string

Reply:
added